tests/expr/fbinary/test-add.py

```diff
@@ -21,10 +21,39 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
 # IN THE SOFTWARE.
 #-------------------------------------------------------------------------------
+import pytest
 from datatable import dt, f
 from tests import assert_equals
 
 
+#-------------------------------------------------------------------------------
+# Errors
+#-------------------------------------------------------------------------------
+
+@pytest.mark.parametrize("type", [dt.obj64])
+def test_add_error_wrong_one_type(type):
```

Comment:
no point in making this parameterized...

Reply:
We may have some new types added, then it will be easier to adjust this test.